src/core/unicode.js

```diff
@@ -1630,8 +1630,29 @@ function reverseIfRtl(chars) {
   return s;
 }
 
+function keepUnifiedIdeograph(str, normalizedUnicodesMap) {
+  // a character id map to a unicode. replace it if it is not unified
+  if (str.length === 1) {
+    if (normalizedUnicodesMap[str]) {
+      return normalizedUnicodesMap[str];
+    }
+    return str;
+  }
+  // a character id map to multiple unicode. remove not unified unicodes
+  var outStr = '';
+  for (var i = 0; i < str.length; i++) {
+    var char = str[i];
+    // if a char is not exist in the map, then it is a normal unicode
+    if (!normalizedUnicodesMap[char]) {
```

Comment:
Like above, using `=== undefined` here is perhaps better since that is what the data structure uses.